geneagrapher/grabber.py

```diff
@@ -24,8 +25,8 @@ def get_record(self, id):
         advisor ids, the mathematician name, the mathematician
         institution, and the year of the mathematician's degree.
         """
-        url = "http://genealogy.math.ndsu.nodak.edu/id.php?id=" + str(id)
-        page = urllib.request.urlopen(url)
+        url = "http://mathgenealogy.org/id.php?id=" + str(id)
+        page = urllib.request.urlopen(url, context=ssl._create_unverified_context())
```

Comment:
I still don't like this, but I believe it's misconfiguration on the server end. Here's some feedback from a couple sites that verify SSL certificates:

From https://www.sslshopper.com/ssl-checker.html#hostname=mathgenealogy.org:

![image](https://user-images.githubusercontent.com/933552/87882023-05fc3d80-c9b2-11ea-9b33-180e1944b572.png)

From https://www.digicert.com/help/:

![image](https://user-images.githubusercontent.com/933552/87882034-20ceb200-c9b2-11ea-967b-09f5b9afdffb.png)

So I think we don't have any alternative at the moment.